packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1599,11 +1599,11 @@ class RenderTransform extends RenderProxyBox {
   RenderTransform({
     @required Matrix4 transform,
     Offset origin,
-    FractionalOffset alignment,
+    Alignment alignment,
     this.transformHitTests: true,
     RenderBox child
   }) : assert(transform != null),
-       assert(alignment == null || (alignment.dx != null && alignment.dy != null)),
+       assert(alignment == null || (alignment.x != null && alignment.y != null)),
```

Comment:
You can remove the second half of this assert since Alignment can't be created with null values.

Reply:
Done.